recipes-dotnet/corefx/corefx_1.1.0.bb

```diff
@@ -34,19 +43,34 @@ do_install() {
 
 	install -d ${target}
 
-	for i in `${src}/Linux.arm.Release/Native/*.so`
+	# Install the stripped binaries, the unstripped are smaller and causes Bus error
+	for i in `ls ${src}/Linux.arm.Release/Native/*.so`
 	do
-		# The non-stripped versions are called *.so.dbg [already-stripped]
-		install -m 0755 ${src}/Linux.arm.Release/Native/${i}.dbg ${target}/${i}
+		install -m 0755 ${i} ${target}
 	done
-	
+
 	for i in ${DLLS}
 	do
 		install -m 0644 ${src}/${i} ${target}
 	done
+
+	# Add link so the runtime can find libuv
+	ln -s /usr/lib/libuv.so.1 ${target}/libuv.so
+
+	# Add DLLs from Microsofts tools distribution
+	for i in ${TOOLS_DLLS}
+	do
+		install -m 0644 ${S}/Tools/${i} ${target}
+	done
 }
 
 FILES_${PN} = "/opt/dotnet"
+TOOLS_DLLS = "\
+  Microsoft.CodeAnalysis.CSharp.dll \
+  Microsoft.CodeAnalysis.dll \
+  Microsoft.CodeAnalysis.VisualBasic.dll \
+  Microsoft.CSharp.dll \
+"
 
```

Comment:
This is Roslyn (not proprietary :-). It's likely this will become available through the `core-setup` and CLI recipes that are the next steps.

It's good to have them already, though.